### PR TITLE
MULTISITE-10348 Basic build-custom skeleton target for subsite specif…

### PR DESCRIPTION
**Pull request by mathiasselleslach**
Providing a basic custom build skeleton target that would allow subsites to specify custom build steps.

### CODE REVIEW

File: build-custom.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project name="My subsite" default="help">
+    <target name="custom-build">
+        <echo message="Running your custom stuff" />
```
Comment (pfrenssen) [inline build-custom.xml]: I would give this target also a description, then it shows up in the list of "important" targets when people do `$ ./bin/phing`.

**Comment by verbruggenalex** [comment]
This should become part of https://github.com/ec-europa/subsite-starterkit/pull/16 where we allow subsites to run a custom composer.json. But should we not move the custom-build.xml also to the resources folder for consistency? Or is it mandatory for the build files to be in the root? @ademarco

File: build.xml [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
     <import file="build.package.xml" />
     <import file="build.test.xml" />
+    <import file="build-custom.xml" />
```
Comment (ademarco) [inline build.xml]: We should probably keep the same pattern as the other files, so I would rename it into `build.custom.xml` and I would move it to `resources/build.custom.xml`.

File: build-custom.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project name="My subsite" default="help">
+    <target
+        name="custom-build"
+        description="You can use this target to perform custom build steps for your subsite. This target is part of both build-dev and build-dist."
+    >
+        <echo message="Running your custom stuff" />
```
Comment (ademarco) [inline build-custom.xml]: Probably a simple `Running custom build` would be more in line with other messages style?

**Comment by ademarco** [comment]
I would actually rebase this branch on #16 and merge it in after #16 gets in, what do you think @verbruggenalex? For more info please refer to https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-12288

File: build-custom.xml (hunk 3 above)
Comment (mathiasselleslach) [inline build-custom.xml]: @ademarco i dont understand this. We do use build-dist and build-dev as such? :)

Comment (ademarco) [inline build-custom.xml]: sorry I was just referring to the message itself, so like:

```diff
-  <echo message="Running your custom stuff" />
+  <echo message="Running custom build" />
```

Just to jet rid of `your custom stuff` which is a little too colloquial IMO. :)

Comment (mathiasselleslach) [inline build-custom.xml]: haha, agreed!
will change!

**Comment by mathiasselleslach** [comment]
@verbruggenalex & @ademarco i would like one of you (or both) to review and merge once approved.
I'd prefer not to rebase, since this is a separate piece of functionality...? :-)

File: README.md [hunk 4]
```diff
@@ -59,6 +59,9 @@ The configuration of the project is managed in 3 `build.properties` files:
   downloaded and copied into the Drupal website.
 * If you have any custom Composer dependencies, declare them in
   `resources/composer.json` and `resources/composer.lock`.
+* If you require custom build steps for your subsite, you are free to use the 
+  `resources/build.custom.xml` phing target. This target is included by default
+   in build-dev & build-dist targets
```
Comment (verbruggenalex) [inline README.md]: Build is good, but documentation is missing a dot :( I know I'm too critical.

**Pull request merged**